package.json

```diff
@@ -41,7 +41,7 @@
     "jquery": "2.2.4",
     "jquery-validation": "1.19.3",
     "magnific-popup": "git+https://github.com/wet-boew/Magnific-Popup.git#1.0.0+keyboard_trap_fix",
-    "mathjax": "2.7.4",
+    "mathjax": "^3.2.0",
```

Comment:
Wouldn't the status quo be better since it sets the minimum version to 3.2.0, while also allowing newer minor versions (3.x)? Try throwing ``^3.2.0`` into the [npm semver version calculator](https://semver.npmjs.com/)'s range field to visualize what it'd represent in practice.

Setting this to the latest major version (e.g. ``3``, ``3.x``, etc) would be kind of misleading since it'd technically indicate that WET's MathJax dependency is compatible with versions 3.0.0-3.1.4 (which we can't guarantee since those older versions have never been tested in WET).